plugin/trino-redshift/pom.xml

```diff
@@ -186,6 +186,7 @@
                         <configuration>
                             <excludes>
                                 <exclude>**/TestRedshiftAutomaticJoinPushdown.java</exclude>
+                                <exclude>**/TestRedshiftConnectorSmokeTest.java</exclude>
```

Comment:
We don't run default profile on CI.
Also, there currently is no profile to run all tests.

Let's make the default profile run all tests (remove any exclusions)
then we need exclusions to define cloud-tests profile

Reply:
I've removed the `default` profile change for `maven-surefire-plugin`.